keyboards/keebio/quefrency/keymaps/marianas/customLogic.c

```diff
@@ -0,0 +1,381 @@
+#include QMK_KEYBOARD_H
+#include "customLogic.h"
+#include "keymap.h"
+#include "keyDefinitions.h"
+#include "relativity.h"
+#include "timer.h"
+
+static int16_t fnTimer = 0;
+
+static uint16_t random_value = 26339;
+
+uint32_t layer_state_set_user(uint32_t state)
+{
+  switch (biton32(state))
+  {
+    case QWERTY:
+      rgblight_mode(9);
```

Comment:
eeek!!

You want to use the noeeprom version.  Otherwise, you're burning through your EEPROM. 

```suggestion
      rgblight_mode_noeeprom(9);
```